guidelines.rst

```diff
@@ -0,0 +1,183 @@
+PEP: 9999
+Title: Guidelines for Python's C API
+Author: Petr Viktorin <encukou@gmail.com>,
+Discussions-To: https://discuss.python.org/c/c-api/30
+Status: Draft
+Type: Process
+Requires: 731
+Created: 20-Nov-2023
+Post-History: XXX
+
+.. pep-banner::
+
+   This is a **incomplete draft**, published for initial discussion.
+   It is likely to be amended substantially.
+
+.. Authors to be added:
+
+        Guido van Rossum <guido@python.org>,
+        Victor Stinner <vstinner@python.org>,
+        Steve Dower <steve.dower@python.org>,
+        Irit Katriel <irit@python.org>
+
+
+Abstract
+========
+
+TODO: Write the abstract last :)
+
+
+About this document
+===================
+
+These guidelines represent the consensus of the C API working group,
+established in :pep:`731` to oversee and coordinate
+the development and maintenance of the Python C API.
+
+It is a living document. It can be changed with the approval of either
+all members of the C API working group, or the Steering Council.
+
+
+High-level goals
+================
```

Comment:
I guess for a PEP, this would be a subheading under the _Specification_ section. Perhaps we should establish a _Speficiation_ heading and put the relevant subheadings under it.

Reply:
This is not a standards track PEP. Is *Specification* appropriate for guidelines?

Reply:
Well, _Specification_ was used for the Standards Track PEP-689, so I guess it can be used for guidelines ;)

Reply:
We also used a _Specification_ section for the Standards Track PEP-687.

Reply:
Right; I just realised this is a _process_ PEP 😄

Reply:
FWIW, it also already says:
> It is a living document. It can be changed with the approval of either
> all members of the C API Working Group, or the Steering Council.